### PR TITLE
Fix problem with non wp-error object:$errors

### PR DESCRIPTION
**Pull request by EarthmanWeb**

### CODE REVIEW

File: slt-force-strong-passwords.php [hunk 1]
```diff
@@ -114,7 +114,18 @@ function slt_fsp_validate_strong_password( $errors, $user_data ) {
...
-	if ( ( false === $password ) || ( $errors->get_error_data("pass") ) ) {
+
+	global $wp_error;  // also check this
+	if (is_wp_error($errors)) { // is this a WP error object?
+		if($errors->get_error_data("pass")){
+			return $errors;
+		}
+	} elseif (is_wp_error($wp_error)) {  // is this a WP error object?
+		if($wp_error->get_error_data("pass")){
+			$errors = $wp_error; // not sure if this is needed...
```
Comment (aaronfay) [inline slt-force-strong-passwords.php]: Why the assignment? `$errors` cannot be used after the `return`...

**Comment by kadamwhite** [comment]
This is tripping us up, too. Since this plugin's installed for all users on WPEngine the surface for this error is very high—a strong +1 from me on this solution, I hope this makes its way to a release soon!

**Comment by boogah** [comment]
1.6.5 — which includes this change — is being sent up to the repo now.

Thanks for the fix @EarthmanWeb! And sorry about the delay!

**Comment by EarthmanWeb** [comment]
Awesome, thanks @boogah

**Pull request merged**